dask-gateway/dask_gateway/gateway.yaml

```diff
@@ -19,3 +19,7 @@ gateway:
   cluster:
     options: {}         # Default options to use when calling ``new_cluster`` or
                         # ``cluster_options``. Values may be template strings.
+
+  httpclient:
+    type: null         # The kind of http client to use. One of {"curl", "simple"}. If not provided, curl is used if pycurl is installed, otherwise simple is used.
+    defaults: null     # A mapping of default parameters to use on every request
```

Comment:
Perhaps add a link to the corresponding tornado docs here? I think http://www.tornadoweb.org/en/stable/httpclient.html#http-client-interfaces would be the right one.

Reply:
Also, please line wrap to ~80 characters